source/exe/main_common.h

```diff
@@ -84,6 +84,7 @@ class MainCommon {
 public:
   MainCommon(int argc, const char* const* argv);
   bool run() { return base_.run(); }
+  Event::Dispatcher& dispatcher() { return base_.server()->dispatcher(); }
```

Comment:
If this is only exposed for test purposes, should we make that explicit with name or comment?

Reply:
Sure.